tools/ark-cli/src/lib/arkApiClient.ts

```diff
@@ -158,13 +158,18 @@ export class ArkApiClient {
   async *createChatCompletionStream(
     params: OpenAI.Chat.Completions.ChatCompletionCreateParams
   ): AsyncIterable<OpenAI.Chat.Completions.ChatCompletionChunk> {
-    const stream = await this.openai.chat.completions.create({
-      ...params,
-      stream: true,
-    });
+    try {
+      const stream = await this.openai.chat.completions.create({
+        ...params,
+        stream: true,
+      });
 
-    for await (const chunk of stream) {
-      yield chunk;
+      for await (const chunk of stream) {
+        yield chunk;
+      }
+    } catch (err) {
```

Comment:
I *think* this will already show a proper error message to the user and kill the CLI so we can remove the try/catch and just a comment